uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveRequestTest.php

```diff
@@ -2335,4 +2335,292 @@ public function testToilToAccrueAmountIsSavedCorrectlyWhenAmountToAccrueIsAFloat
     sort($toilAmounts);
     $this->assertEquals($toilAmounts, $toilToAccrueAmounts);
   }
+
+  /**
+   * @expectedException CRM_HRLeaveAndAbsences_Exception_InvalidLeaveRequestException
+   * @expectedExceptionMessage The TOIL amount plus all approved TOIL for current period is greater than the maximum for this Absence Type
+   */
+  public function testUpdatingToilThrowsExceptionWhenUpdatingApprovedToilWithAToilAmountGreaterThanWhatWasInitiallyApprovedAndTotalToilToBeAccruedIsGreaterThanMaximumForTheAbsenceType() {
```

Comment:
BTW using `phpuint4 --testdox` makes long names useful as you can use that to view all tests in a class in readable English.